### ISSUE TITLE
[Feature] Please make this more user friendly to install. II've spent  half hour following your instructions for nothing. 

### ISSUE THREAD

**Issue by ashtonvalasko** [exislow/tidal-dl-ng#88]
### Current Situation

None of. the instructions. work. on windows. 

### Please make the code INCLUDING all pre-reqs, there's too much going on for  a  final product

**Comment by NoDowt** [comment]
There is the pre-compiled Exe version if you want easy. That’s all I use & works great.

**Comment by exislow** [comment]
@ashtonvalasko I do not use windows, so I cannot help you with that. And according to the lack of any further information and logs / screenshots nobody else is able to help you here I guess. Please put a little more effort in creating issues.

**Comment by NoDowt** [comment]
I did briefly try the native python version before the Exe was available & couldn’t get it working either… but it was also my first time trying to use python.
I never got a chance to troubleshoot (as you made the Exe available, so no need) but suspect it was likely related to the version of Python I had installed… don’t think you mentioned a recommended version at the time, but now I see 3.11 is the recommended.

I might give it another shot when I’m back from holidays.
